src/SDKs/KeyVault/data-plane/Microsoft.Azure.KeyVault.TestFramework/Microsoft.Azure.KeyVault.TestFramework.csproj

```diff
@@ -5,6 +5,7 @@
     <AssemblyTitle>Microsoft Azure Key Vault Test Framework</AssemblyTitle>
     <IsTestHelperLibrary>true</IsTestHelperLibrary>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <AssemblyOriginatorKeyFile>C:\Git\azure-sdk-for-net\eng\AzSdkTestLibKey.snk</AssemblyOriginatorKeyFile>
```

Comment:
Please remove this should not be necessary.

Reply:
Ok